examples/pybullet_industrial_intro.py

```diff
@@ -0,0 +1,167 @@
+
+def build_path(x_data, y_data,number_of_steps):
+    z_data=[]
+    
+    i=0
+    path_list=[]
+    while i<len(x_data):
+        if i<len(x_data)-1:
+            sub_path=pi.linear_interpolation(np.array([2.5-x_data[i], y_data[i]-1, height]),np.array([2.5-x_data[i+1], y_data[i+1]-1, height]),number_of_steps)
+        if i==len(x_data)-1:
+            sub_path=pi.linear_interpolation(np.array([2.5-x_data[i], y_data[i]-1, height]),np.array([2.5-x_data[0], y_data[0]-1, height]),number_of_steps)
+        path_list.append(sub_path)
+        i+=1
+    path=path_list[0]
+    i=1
+    while i<len(path_list):
+        path.append(path_list[i])
+        i+=1
+        
+    path.draw()
+    return path
+
+def scale_paths(path):
+    
+    path_copy_list=[]
+    max_x=np.max(path.positions[0])
+    min_x=np.min(path.positions[0])
+    width_x=np.abs(max_x-min_x)
+    
+    max_y=np.max(path.positions[1])
+    min_y=np.min(path.positions[1])
+    width_y=np.abs(max_y-min_y)
+
+    #assuming that path stays at the same height
+    height=np.mean(path.positions[2])
+
+    scale_list=[0.2,0.4,0.6]
+    center_point=np.array([min_x+width_x/2, min_y+width_y/2, height])
+    for scale in scale_list:
+        
+        path_copy=path
+        path_copy.translate(-center_point)
+        
+        path_copy.positions*=scale
+        path_copy.translate(center_point)
+        
+        path_copy.draw()
+        path_copy_list.append(path_copy) 
+    return path_copy_list
+
+def build_smaller_paths(path, scale):
+    x_list=[]
+    y_list=[]
+    z_list=[]
+    
+    for x,y,z in zip(path.positions[0],path.positions[1],path.positions[2]):
+        x_list.append(x)
+        y_list.append(y)
+        z_list.append(z)
+    i=0
+    new_x_list=[]
+    new_y_list=[]
+    new_z_list=[]
+    while i<len(x_list)-1:
+        new_z_list.append(height)
+    i=0
+    while i<len(x_list)-1:
+        point=np.array([x_list[i], y_list[i], z_list[i]])
+        diff_vector=point-np.array([x_list[i+1], y_list[i+1], z_list[i+1]])
+        theta=np.pi /4
+        x=np.cos(theta)*diff_vector[0]-np.sin(theta)*diff_vector[1]
+        y=np.sin(theta)*diff_vector[0]+np.cos(theta)*diff_vector[1]
+        new_x_list.append(x)
+        new_y_list.append(y)
+        i+=1
+    new_points=np.array([new_x_list, new_y_list, new_z_list])
+    new_path=pi.ToolPath(new_points)
+    new_path.draw()
+
+    return new_path
+    
+
+
+
+if __name__ == "__main__":
+    dirname = os.path.dirname(__file__)
+    urdf_file1 = os.path.join(dirname,
+                              'robot_descriptions', 'comau_NJ290_3-0_m.urdf')
+    urdf_file2 = os.path.join(dirname,
+                              'robot_descriptions', 'milling_head.urdf')
+
+    physics_client = p.connect(p.GUI)
+    p.setPhysicsEngineParameter(numSolverIterations=5000)
+
+    p.setAdditionalSearchPath(pybullet_data.getDataPath())
```

Comment:
While the monastery looks nice it does not realy work with the wbk branding and should be removed